common/pistage.go

```diff
@@ -35,7 +34,7 @@ func (p *Pistage) init() {
 }
 
 func (p *Pistage) Name() string {
```

Comment:
Can deprecate `Name()` method and replace it with identifier now that namespace is discarded.